src/state/OptionsState.cpp

```diff
@@ -401,7 +401,13 @@ InputOptionsState::InputOptionsState()
 	mSetKeyboard = 0;
 	mOptionConfig.loadFile("inputconfig.xml");
 	//left data:
+#ifdef __SWITCH__
+	mLeftDevice = "joystick";
+	mRightDevice = "joystick";
```

Comment:
right device config in "//left data:" area. Please move right device config in "//right data:" area.